docs/guides/high-availability/partials/network-latency.adoc

```diff
@@ -0,0 +1,7 @@
+// See https://www.cloudping.co/ for an overview of AWS latencies between AZs
+
+A P50 round-trip latency of less than 5 ms is suggested and below 10 ms is strongly recommended, together with a full ownership of the network between the zones.
```

Comment:
It's not clear to me what you mean by "full ownership of the network between zones".